scripts/get-test-pass-rate.js

```diff
@@ -0,0 +1,55 @@
+// This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used 
+// to get timeline and metrics data so that the test pass rate can be calculated in a later step.
```

Comment:
I would also recommend documenting the prerequisites for running this script here at the top. E.g. the variables we expect to have been set / what they mean.